vuepress/docs/next/tutorials/devops/entando-de-app.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# Configure the Entando App Context
+This tutorial describes how to modify the [EntandoApp CRD](../../docs/consume/entandoapp-cr.md) to customize the context path of an Entando App Engine deployment. 
+
+## Prerequisites
+- An existing deployment of an Entando App or the ability to create one.
+    - If you haven't created a deployment or don't have a YAML file for an Entando deployment, follow the [Quickstart instructions](../../docs/getting-started/README.md).
+
+## Remove or Replace entando-de-app
+
+The web context of the App Engine currently defaults to **/entando-de-app**, which is visible in the URL. Follow the steps below to remove or replace this value.
```

Comment:
"web context" -> "ingress path"

Reply:
@nshaw should we change the language in the table on the linked page?

Reply:
@Lyd1aCla1r3 Maybe I'm missing it. The getting-started/README seems okay to me

Reply:
@nshaw sorry - this page https://developer.entando.com/next/docs/consume/entandoapp-cr.html - the table entry for ingressPath

Reply:
@Lyd1aCla1r3 hah, that may have been one I was looking at. Yeah, I think we should clean that up. And it could be useful to add it to the navigation under consume since it's an important reference and hard to find right now.